packages/dataviews/src/stories/dataviews.fixtures.tsx

```diff
@@ -812,6 +825,386 @@ export const actions: Action< SpaceObject >[] = [
+		type: 'payment',
+		categories: [ 'Payment', 'Credit Card' ],
+		date: '2025-01-15',
+		datetime: '2025-01-15T09:23:47Z',
+		email: 'alice.wonderland@fictional-store.test',
+		orderNumber: '#2502',
+	},
+	{
+		id: 3,
+		name: {
+			title: 'Order Confirmation Sent',
+			description:
+				'Order confirmation #1259 sent to alice.wonderland@fictional-store.test',
+		},
+		type: 'email',
+		categories: [ 'Email', 'Communication' ],
+		date: '2025-01-15',
+		datetime: '2025-01-15T09:24:02Z',
+		email: 'alice.wonderland@fictional-store.test',
+		orderNumber: '#2502',
+	},
+	{
+		id: 4,
+		name: {
+			title: 'Private Note Added',
+			description:
+				'Buyer has requested to wait a couple days to send the order, they are not going to be home until the weekend.',
+		},
+		type: 'note',
+		categories: [ 'Note', 'Internal' ],
+		date: '2025-01-15',
+		datetime: '2025-01-15T14:32:18Z',
+		email: 'store.admin@fictional-store.test',
+		orderNumber: '#2502',
+	},
+	{
+		id: 5,
+		name: {
+			title: 'Status Changed to Processing',
+			description:
+				'Order status automatically changed from pending to processing.',
+		},
+		type: 'status',
+		categories: [ 'Order', 'Status' ],
+		date: '2025-01-15',
+		datetime: '2025-01-15T14:35:00Z',
+		orderNumber: '#2502',
+	},
+	{
+		id: 6,
+		name: {
+			title: 'Customer Note Added',
+			description:
+				'Customer added note: "Please leave the package at the back door. Thank you!"',
+		},
+		type: 'note',
+		categories: [ 'Note', 'Customer' ],
+		date: '2025-01-16',
+		datetime: '2025-01-16T08:15:42Z',
+		email: 'alice.wonderland@fictional-store.test',
+		orderNumber: '#2502',
+	},
+	{
+		id: 7,
+		name: {
+			title: 'Items Packed',
+			description:
+				'All items have been packed and are ready for shipment.',
+		},
+		type: 'fulfillment',
+		categories: [ 'Fulfillment', 'Warehouse' ],
+		date: '2025-01-18',
+		datetime: '2025-01-18T10:22:33Z',
+		orderNumber: '#2502',
+	},
+	{
+		id: 8,
+		name: {
+			title: 'Shipping Label Created',
+			description:
+				'Shipping label created with USPS Priority Mail. Tracking #9400111899562854217803',
+		},
+		type: 'shipping',
+		categories: [ 'Shipping', 'USPS' ],
+		date: '2025-01-18',
+		datetime: '2025-01-18T11:05:14Z',
+		orderNumber: '#2502',
+	},
+	{
+		id: 9,
+		name: {
+			title: 'Order Shipped',
+			description:
+				'Order has been shipped via USPS Priority Mail. Expected delivery: Jan 20, 2025',
+		},
+		type: 'shipping',
+		categories: [ 'Shipping', 'Status' ],
+		date: '2025-01-18',
+		datetime: '2025-01-18T16:42:09Z',
+		email: 'alice.wonderland@fictional-store.test',
+		orderNumber: '#2502',
+	},
+	{
+		id: 10,
+		name: {
+			title: 'Shipment Notification Sent',
+			description:
+				'Shipment notification email with tracking information sent to customer.',
+		},
+		type: 'email',
+		categories: [ 'Email', 'Communication' ],
+		date: '2025-01-18',
+		datetime: '2025-01-18T16:42:25Z',
+		email: 'alice.wonderland@fictional-store.test',
+		orderNumber: '#2502',
```

Comment:
It'd be nice to introduce different values in orderNumber and email, for the filters to display variety.